lamindb_setup/_init_vault.py

```diff
@@ -0,0 +1,41 @@
+from typing import Optional
+
+from pydantic import PostgresDsn
+
+from ._settings import settings
+from .dev._hub_utils import LaminDsnModel
+from .dev.cloud_sqlite_locker import unlock_cloud_sqlite_upon_exception
+
+
+@unlock_cloud_sqlite_upon_exception(ignore_prev_locker=True)
```

Comment:
We don't need the decorator here, because it's only relevant for SQLite instances!

Reply:
Ok